Cargo.toml

```diff
@@ -40,6 +40,7 @@ members = [
     "megazords/ios/rust",
     "testing/sync-test",
     "tools/protobuf-gen",
+    "tools/uniffi-builtin-bindgen",
```

Comment:
'builtin' kinda jarred with me - 'embedded' sounds a bit better, but I'm not sure why as that's pretty much a synonym. I think 'builtin' made me thing 'built in to uniffi'. No strong opinion here, but I thought I'd open a bikeshed.

Reply:
Fair point. I invested about five seconds worth of effort into the name here, and went with "builtin" entirely by analogy to `features=["builtin-bindgen"]` which you need to add to the config of each crate. I'd be fine with "embedded", and it might also help to re-order to words to something like "embedded-uniffi-bindgen" or "builtin-uniffi-bindgen" so that it's less suggestive of being built in to uniffi.